cartography/intel/aws/iam.py

```diff
@@ -80,7 +80,14 @@ def get_user_policy_data(boto3_session, user_list):
         name = user["UserName"]
         arn = user["Arn"]
         resource_user = resource_client.User(name)
-        policies[arn] = {p.name: p.policy_document["Statement"] for p in resource_user.policies.all()}
+        try:
+            policies[arn] = {p.name: p.policy_document["Statement"] for p in resource_user.policies.all()}
+        except resource_client.meta.client.exceptions.NoSuchEntityException:
+            logger.warning(
+                "Could not get user's policies: %s.policies.all() "
```

Comment:
Minor point: this log message for my username would look like `Could not get user's policies: achantavy.policies.all() failed with NoSuchEntityException; skipping.`. Is this intended? Something like `Could not get policies for user achantavy due to NoSuchEntityException; skipping.` might make more sense.

Reply:
The full error message is as follows: 

```
WARNING:cartography.intel.aws.iam:Could not get user's policies: iam.User(name='USERNAME').policies.all() failed with NoSuchEntityException; skipping.
```

I'll change it to produce the suggest: 

```
Could not get policies for user achantavy due to NoSuchEntityException; skipping.
```